tidy3d/plugins/design/result.py

```diff
@@ -61,23 +62,18 @@ class Result(Tidy3dBaseModel):
         description="Source code for the function evaluated in the parameter sweep.",
     )
 
-    task_ids: Tuple[Tuple[str, ...], ...] = pd.Field(
+    task_ids: list[str] = pd.Field(
```

Comment:
we use `tuple[str, ...]` because it's immutable.

Reply:
That's fair, will address this once we settle on the name generator approach